study5_200303/jihye.js

```diff
@@ -0,0 +1,57 @@
+/*
+1. 가운데 글자 가져오기
+단어 s의 가운데 글자를 반환하는 함수, solution을 만들어 보세요. 단어의 길이가 짝수라면 가운데 두글자를 반환하면 됩니다.
+재한사항 s는 길이가 1 이상, 100이하인 스트링입니다.
+*/
+function solution1(s) {
+    var answer = '';
+    
+    if (s.length % 2 === 0) {
+      let num;
+      num = s.length/2;
+      //console.log(num);
+      answer = s[num-1] + s[num];
+      //console.log(answer);
+    } else if (s.length % 2 === 1) {
+      let num;
+      num = Math.floor(s.length/2);
+      answer = s[num];
+      //console.log(answer);
+    }
```

Comment:
early return 을 사용해보셔요.

```suggestion
    const midPosition = Math.floor(s.length / 2);
    if (s.length % 2 === 0) {
      return s[midPosition-1] + s[midPosition];
    }
    
    return s[midPosition];
```

Reply:
let보다는 가능하다면 const로 선언하시는게 좋아요 :)